Formula/orientdb.rb

```diff
@@ -48,23 +43,33 @@ def post_install
     (var/"log/orientdb").mkpath
     touch "#{var}/log/orientdb/orientdb.err"
     touch "#{var}/log/orientdb/orientdb.log"
+
+    ENV["ORIENTDB_ROOT_PASSWORD"] = "orientdb"
+    system "#{bin}/orientdb", "start"
+    sleep 3
+    system "#{bin}/orientdb", "stop"
   end
 
-  def caveats
-    "Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`."
+  def caveats; <<-EOS.undent
+      The OrientDB root password was set to 'orientdb'. Follow this to reset it:
+      http://orientdb.com/docs/2.2/Server-Security.html#restoring-the-servers-user-root
+
+      Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`.
```

Comment:
Would be good to just remove. It's not Homebrew specific information so the user can figure it out themselves or from the official docs.